tests/test_configuration.py

```diff
@@ -0,0 +1,45 @@
+#
+# Sonar Scanner Python
+# Copyright (C) 2011-2023 SonarSource SA.
+# mailto:info AT sonarsource DOT com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+#
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+import unittest
+from unittest.mock import patch, Mock
+from py_sonar_scanner.configuration import Configuration
+
+
+class TestConfiguration(unittest.TestCase):
+    @patch("py_sonar_scanner.configuration.sys")
+    def test_argument_parsing_empty_toml(self, mock_sys):
```

Comment:
One case that would be worth adding is the one where no extra argument is provided.

Reply:
Yes, this makes sense. I added this in the last commit.